node/src/node_configurator/configurator.rs

```diff
@@ -2647,6 +2657,7 @@ mod tests {
                 "0x0123456789012345678901234567890123456789".to_string(),
             )))
             .start_block_result(Ok(3456))
+            .max_block_count_result(Ok(100000))
```

Comment:
I've seen a number of tests about the `Some()` side of the `max_block_count_opt` retrieval; is there at least one test of the `None` side?